app/db/migrations/versions/fdf8821871d7_main_tables.py

```diff
@@ -193,7 +195,39 @@ def create_commentaries_table() -> None:
         EXECUTE PROCEDURE update_updated_at_column();
         """
     )
+def create_new_user(username, email, password, admin = False) -> None:
```

Comment:
In the future, replace this direct SQL functions with using the already created files